### PR TITLE
Dram hit ratio algorithm

### PR DESCRIPTION
**Pull request by mmucek95**

### CODE REVIEW

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+from wca.metrics import Metric, MetricType
+from wca.scheduler.algorithms.base import BaseAlgorithm, QueryDataProviderInfo
+from wca.scheduler.metrics import MetricName
+from wca.scheduler.types import NodeName
+
+log = logging.getLogger(__name__)
+
+
+THRESHOLD = 0.99
```
Comment (Creatone) [inline wca/scheduler/algorithms/dram_hit_ratio_provision.py]: The threshold could be configurable. ```0.99``` can be a default value for this parameter.

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+class DramHitRatioProvision(BaseAlgorithm):
+    def app_fit_node(self, node_name: NodeName, app_name: str,
```
Comment (Creatone) [inline wca/scheduler/algorithms/dram_hit_ratio_provision.py]: There are no tests for this method. 
It's the major feature of your PR, so in my opinion, this should be tested properly.

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+from wca.scheduler.metrics import MetricName
+from wca.scheduler.types import NodeName
+
+log = logging.getLogger(__name__)
+
+
+THRESHOLD = 0.99
+
+
+class DramHitRatioProvision(BaseAlgorithm):
```
Comment (Creatone) [inline wca/scheduler/algorithms/dram_hit_ratio_provision.py]: There can be a little comment what this algorithm does.

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py (hunk 2 above)
Comment (mmucek95) [reply wca/scheduler/algorithms/dram_hit_ratio_provision.py]: Done

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py (hunk 1 above)
Comment (mmucek95) [reply wca/scheduler/algorithms/dram_hit_ratio_provision.py]: Done

File: wca/scheduler/algorithms/dram_hit_ratio_provision.py (hunk 3 above)
Comment (mmucek95) [reply wca/scheduler/algorithms/dram_hit_ratio_provision.py]: Done

**Pull request merged**